src/new-components/select-field/select-field.module.css

```diff
@@ -33,6 +33,7 @@
     appearance: none;
     box-sizing: border-box;
     width: 100%;
+    color: var(--reactis-content-primary);
```

Comment:
```suggestion
    color: var(--reactist-content-primary);
```

Reply:
This reminds me of re-considering using https://github.com/seek-oss/vanilla-extract internally. With it you get static analysis of css variables, builds fail if you use invalid css variable names, just as it happens for JS today.

I probably will open a thread about it.